Libraries/test/Amazon.Lambda.RuntimeSupport.Tests/Amazon.Lambda.RuntimeSupport.UnitTests/FileDescriptorLogStreamTests.cs

```diff
@@ -16,7 +16,7 @@ public class FileDescriptorLogStreamTests
 
         private static readonly byte[] ExpectedMagicBytes =
         {
-            0xA5, 0x5A, 0x00, 0x01
+            0xA5, 0x5A, 0x00, 0x03
```

Comment:
GitHub won't let me comment on it directly, but this test is failing for me locally with this branch checked out:

```
Amazon.Lambda.RuntimeSupport.UnitTests.FileDescriptorLogStreamTests.MultilineLoggingInSingleLogEntryWithTlvFormat
   Source: FileDescriptorLogStreamTests.cs line 23
   Duration: 33 ms

  Message: 
Assert.Equal() Failure
Expected: Byte[] [0, 0, 0, 41]
Actual:   Byte[] [83, 171, 221, 67]

  Stack Trace: 
FileDescriptorLogStreamTests.AssertHeaderBytes(Byte[] buf, Byte[] expectedLengthBytes) line 190
FileDescriptorLogStreamTests.MultilineLoggingInSingleLogEntryWithTlvFormat() line 66
```

Reply:
Moved this to follow-up https://github.com/aws/aws-lambda-dotnet/pull/1409